### ISSUE TITLE
SOC in calculations

### ISSUE THREAD

**Issue by simonganne01** [pyscf/pyscf#2400]
Is there a way to include spin-orbit coupling in the kernel calculations of for example the KUKS object?

**Comment by sunqm** [comment]
No. Only KGKS supports spin-orbit contributions through the `mf.x2c()` method. However, the default x2c for pbc suffers from stability issues, please the discussions in https://github.com/pyscf/pyscf/issues/2174 . You might need to use the atomic variant by setting `mf.x2c.approx = 'ATOM1E'`
